en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-connector-configuration.md

```diff
@@ -0,0 +1,30 @@
+# Setting up ISO8583 Connector 
+
+ISO8583 is an international standard for financial transaction card originated interchange messaging. It is the International Organization for Standardization standard for systems that exchange electronic transactions initiated by cardholders using payment cards.
+
+Typically, whenever we use a credit card, debit card, or ATM card, the data travels from one system to another system. A card-based transaction typically needs to travel between a number of systems.
+ISO 8593 is Financial Transaction card originated, Interchange message specifications adopted by wide segment of Payment Industry (including Acquirers and Issuers).
```

Comment:
The "Financial Transaction card originated," part doesn't make much sense here. Can we reword that? 

Also change to ISO8583.

Reply:
Updated.